app/helpers/form_helper.rb

```diff
@@ -375,4 +375,9 @@ def field(f, attr, options = {})
       end
     end
   end
+
+  def internetexplorer_multipart_form_data_fix(f)
```

Comment:
No need to pass the form object (`f`) in here. Also name is a bit long, how about `ie_multipart_fix` or something like that?